### PR TITLE
Features/volunteer checklist

### PR DESCRIPTION
**Pull request by chrysalag**
# [Story in Trello](https://trello.com/c/HfAePdKp/65-new-checklist-on-volunteers)

### Done for approval?

* [x] Acceptance criteria are met
* [x] Code quality checks are green
* ~[ ] Readme updated if needed~
* [x] Story under test
* ~[ ] Mobile works~
* ~[ ] Seeds created~
* [x] Translated
* [x] Code is reviewed

### Done after the merge?

* [ ] Deployed to staging after the merge
* [ ] Tested manually on staging

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -104,15 +104,18 @@ de:
...
         adults: Erwachsene
         avatar: Foto
+        bank_account: Kontodaten eingetragen
         birth_year: Jahrgang
         children: Kinder
         city: Stadt
         created_at: Erstellt am
         creative: Kreativ
         culture: Kultur
+        doc_sent: Dossier Freiwillige engagiert verschick
```
Comment (nuritnt) [inline config/locales/de.yml]: verschick**t**

File: config/locales/de.yml [hunk 2]
```diff
@@ -264,6 +269,7 @@ de:
   change_my_password: Ihr Passwort ändern
   change_password: Ihr Passwort ändern
   changed_by: Geändert durch
+  checklist: Abhak-Liste
```
Comment (nuritnt) [inline config/locales/de.yml]: "Abhak Liste" sounds wierd for me, allthough I know she called it like this in the Mail, but what do you think about Checkliste ?

File: test/policies/volunteer_policy_test.rb [hunk 3]
```diff
@@ -42,4 +42,16 @@ def setup
   test 'Seeking clients: department manager cannot see volunteers seeking clients' do
     refute_permit @department_manager, Volunteer, 'seeking_clients?'
   end
+
+  test "Checklist: superadmin can see volunteer's checklist" do
+    assert_permit @superadmin, Volunteer, 'checklist?'
+  end
+
+  test "Checklist: social worker can see volunteer's checklist" do
```
Comment (nuritnt) [inline test/policies/volunteer_policy_test.rb]: social worker **cannot** see volunteer's checklist

**Comment by chrysalag** [comment]
@nuritnt It would be nice to have, but it is not so easily testable. If they ask for a nicer view we can do it then.

**Comment by chrysalag** [comment]
Travis failed because a test randomly fails (is a reported issue).

**Pull request merged**